### PR TITLE
[ADD]partner_ref_sequence

### PR DESCRIPTION
**Pull request by smangukiya**
@max3903 : Please review it. Thanks

### CODE REVIEW

File: partner_ref_sequence/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    "author": "Open Source Integrators, Odoo Community Association (OCA)",
+    "summary": "Generate Automatic customer reference number",
+    "category": "Customers",
+    "maintainer": "Open Source Integrators",
+    "website": "http://www.opensourceintegrators.com",
+    "depends": ["base"],
+    "data": [
+        "data/ir_sequence_data.xml",
+    ],
+    "installable": True,
```
Comment (max3903) [inline partner_ref_sequence/__manifest__.py:17]: @smangukiya You are missing the development_status and the maintainers keys. Website should be https://github.com/OCA/partner-contact

File: partner_ref_sequence/models/res_partner.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# Copyright (C) 2018 - TODAY, Open Source Integrators
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, models, fields
+
+
+class ResPartner(models.Model):
+    _inherit = 'res.partner'
+    _description = 'Add customer to customer number'
```
Comment (max3903) [inline partner_ref_sequence/models/res_partner.py]: No need to reset the description

File: partner_ref_sequence/readme/CONFIGURE.rst [hunk 3]
```diff
@@ -0,0 +1 @@
+* Install the module
```
Comment (max3903) [inline partner_ref_sequence/readme/CONFIGURE.rst]: To be removed.

File: partner_ref_sequence/readme/ROADMAP.rst [hunk 4]
```diff
@@ -0,0 +1,2 @@
+The roadmap of the Partner contanct is documented on
```
Comment (max3903) [inline partner_ref_sequence/readme/ROADMAP.rst]: `partner-contact`

File: partner_ref_sequence/readme/USAGE.rst [hunk 5]
```diff
@@ -0,0 +1 @@
+* No specific steps for usage
```
Comment (max3903) [inline partner_ref_sequence/readme/USAGE.rst]:
```
To use this module:

* Go to Contacts
* Create a partner
* The Customer Number field is automatically set with the next number in the sequence.

File: partner_ref_sequence/readme/DESCRIPTION.rst [hunk 6]
```diff
@@ -0,0 +1,2 @@
+This module will introduce automatic integer number generation for customer.
+And fill in ref field
```
Comment (max3903) [inline partner_ref_sequence/readme/DESCRIPTION.rst]: `This module provides a sequence to generate an automatic number to set the customer number.`

**Review by max3903 — CHANGES_REQUESTED** [review]

**Comment by pedrobaeza** [comment]
Isn't this module the same as https://github.com/OCA/partner-contact/tree/11.0/base_partner_sequence?

**Comment by smangukiya** [comment]
@max3903 : Source has been improved with all your comments. Thanks

**Comment by smangukiya** [comment]
> Isn't this module the same as https://github.com/OCA/partner-contact/tree/11.0/base_partner_sequence?

@pedrobaeza : Yes, it seems with only difference that it will work for customer only. @max3903 : correct me if I am missing here. Thanks

**Comment by pedrobaeza** [comment]
Then better to adapt that one and possibly expand it in a configurable way.

**Comment by max3903** [comment]
Our customer is fine with the current feature set of base_partner_sequence.

**Pull request closed without merging**